docs/source/datasets/index.rst

```diff
@@ -34,88 +35,140 @@ an individual simulant, with the columns representing different simulant attribu
 US Decennial Census
 -------------------
 The Decennial Census dataset is a simulated enumeration of the US Census Bureau's Decennial Census Survey. The years
-that have been simulated are 2020, 2030, and 2040.
+that have been simulated are 2020, 2030, and 2040. To find out more about the survey, please visit the Decennial Census
+`homepage <https://www.census.gov/programs-surveys/decennial-census.html>`_.   
 
 The following simulant attributes are included in this dataset:
 
 .. list-table:: **Simulant attributes**
    :header-rows: 1
 
    * - Attribute Name
-     - Column Name    
+     - Column Name
+     - Notes    
    * - Unique simulant ID
      - :code:`simulant_id`
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - First name
      - :code:`first_name`
+     - 
    * - Middle initial
      - :code:`middle_initial`
+     - 
    * - Last name
      - :code:`last_name`
+     - 
    * - Age
-     - :code:`age`  
+     - :code:`age` 
+     - Rounded down to an integer. 
    * - Date of birth
      - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
    * - Physical address street number
      - :code:`street_number`
+     - 
    * - Physical address street name
      - :code:`street_name`
+     - 
    * - Physical address unit
      - :code:`unit_number`
+     - 
    * - Physical address city
-     - :code:`city`    
+     - :code:`city` 
+     -    
    * - Physical address state
      - :code:`state`  
+     - 
    * - Physical address ZIP code
      - :code:`zipcode`
+     - 
    * - Relationship to person 1 (head of household)
```

Comment:
Can we make this "reference person" everywhere, and reduce the cognitive load of remembering the synonyms?